python/applications/ranking/scripts/run_create_dataset.py

```diff
@@ -0,0 +1,19 @@
+# Run from ml4ir/python directory as: python3 applications/ranking/scripts/run_create_dataset.py
+
+from applications.ranking.data.scripts.create_dataset import create_dataset
+
+
+def main():
```

Comment:
@calliebradley Why not add the `main()` to `create_dataset.py` itself?

Reply:
Oh. This is under `ranking/scripts/..`. I actually think it makes sense to be one single self contained python module under `ranking/data/scripts/create_dataset.py`.

Reply:
@lastmansleeping Yeah, I don't mind moving it. I thought we talked about this in our first syncup, that I built the run example separately when I built the create_dataset module as a class callable from pipeline.py (similar to NER's search assistant modeling process). I'll move it for now and we can always change it in the future.